src/unix/fs.c

```diff
@@ -143,6 +143,18 @@ extern char *mkdtemp(char *template); /* See issue #740 on AIX < 7 */
   while (0)
 
 
+static int uv__fs_close(int fd) {
+  int rc;
+
+  rc = close(fd);
+  if (rc == -1)
```

Comment:
Any reason not to combine the two `if` statements?

Reply:
Personal preference, really. I find two if statements easier to read than `(c1 && (c2 || c3))`.